src/main/jbake/content/docs/user/getting-started.adoc

```diff
@@ -81,10 +81,39 @@ Status: Image is up to date for hawkular/hawkular:latest #<2>
 <1> We ask for the latest build
 <2> In this case the local one was already the latest, so we are good here.
 
-
 === Working on Hawkular source
 
 If you are interested in hacking on Hawkular, or building it from source, then check out
 link:/docs/dev/development.html[Development resources]
 
+== Install a Wildfly monitoring agent
+
+If you wish to monitor a WildFly instance,  you can do so with the following steps:
+
+. Download the WildFly module from http://download.jboss.org/hawkular/wildfly-monitor/0.0.2/hawkular-monitor-0.0.2-module.zip[here].
+. Unzip the file inside the `modules` directory of  your WildFly 8.2 instance
+. Add the following subsystem declaration in the `standalone/configuration/standalone.xml` file of your WildFly instance, the `url` of the storage adapter should be the URL of your Hawkular server.
+
+[source,xml]
+----
+ <subsystem enabled="true" apiJndiName="java:global/hawkular/agent/monitor/api" numMetricSchedulerThreads="3" numAvailSchedulerThreads="3" xmlns="urn:org.hawkular.agent.monitor:monitor:1.0">
+            <storage-adapter type="HAWKULAR" url="http://localhost:8080" context="/hawkular-bus/message/" restContext="/hawkular-metrics/"/>
+            <diagnostics enabled="true" reportTo="LOG" interval="5" timeUnits="minutes"/>
+            <metric-set-dmr name="platform" enabled="true">
+                <metric-dmr name="heap.used" interval="30" timeUnits="seconds" resource="/core-service=platform-mbean/type=memory" attribute="heap-memory-usage#used"/>
+                <metric-dmr name="heap.committed" interval="1" timeUnits="minutes" resource="/core-service=platform-mbean/type=memory" attribute="heap-memory-usage#committed"/>
+                <metric-dmr name="non-heap.used" interval="30" timeUnits="seconds" resource="/core-service=platform-mbean/type=memory" attribute="non-heap-memory-usage#used"/>
+                <metric-dmr name="non-heap.committed" interval="1" timeUnits="minutes" resource="/core-service=platform-mbean/type=memory" attribute="non-heap-memory-usage#committed"/>
+                <metric-dmr name="thread-count" interval="2" timeUnits="minutes" resource="/core-service=platform-mbean/type=threading" attribute="thread-count"/>
+            </metric-set-dmr>
+            <avail-set-dmr name="server-availabilities" enabled="true">
+                <avail-dmr name="app-server" interval="30" timeUnits="seconds" resource="/" attribute="server-state" upRegex="run.*"/>
+                <avail-dmr name="hawkular-bus-broker" interval="1" timeUnits="minutes" resource="/subsystem=hawkular-bus-broker"/>
+            </avail-set-dmr>
+            <managed-servers>
+                <local-dmr enabled="true" metricSets="platform" availSets="server-availabilities"/>
+            </managed-servers>
+        </subsystem>
```

Comment:
we now have resource types that need to be defined, so we need to provide &lt;resource-type-set-dmr> definitions and use those in &lt;managed-servers>. I would remove all the metric-set, avail-set, and managed-server elements here and replace with the following:

`            <metric-set-dmr name="WildFly Memory Metrics" enabled="true">
                <metric-dmr name="Heap Used" interval="30" timeUnits="seconds" metricUnits="bytes" path="/core-service=platform-mbean/type=memory" attribute="heap-memory-usage#used"/>
                <metric-dmr name="Heap Committed" interval="1" timeUnits="minutes" path="/core-service=platform-mbean/type=memory" attribute="heap-memory-usage#committed"/>
                <metric-dmr name="NonHeap Used" interval="30" timeUnits="seconds" path="/core-service=platform-mbean/type=memory" attribute="non-heap-memory-usage#used"/>
                <metric-dmr name="NonHeap Committed" interval="1" timeUnits="minutes" path="/core-service=platform-mbean/type=memory" attribute="non-heap-memory-usage#committed"/>
            </metric-set-dmr>
            <metric-set-dmr name="WildFly Threading Metrics" enabled="true">
                <metric-dmr name="Thread Count" interval="2" timeUnits="minutes" metricUnits="none" path="/core-service=platform-mbean/type=threading" attribute="thread-count"/>
            </metric-set-dmr>

            <avail-set-dmr name="Server Availability" enabled="true">
                <avail-dmr name="App Server" interval="30" timeUnits="seconds" path="/" attribute="server-state" upRegex="run.*"/>
            </avail-set-dmr>

            <resource-type-set-dmr name="Main" enabled="true">
                <resource-type-dmr name="WildFly Server" resourceNameTemplate="WildFly Server" path="/" metricSets="WildFly Memory Metrics,WildFly Threading Metrics" availSets="Server Availability"/>
            </resource-type-set-dmr>

            <managed-servers>
                <local-dmr enabled="true" resourceTypeSets="Main"/>
            </managed-servers>`